modules/effects/arrow-effects-data/src/test/kotlin/arrow/effects/EffectsSuspendDSLTests.kt

```diff
@@ -281,7 +281,7 @@ class EffectsSuspendDSLTests : UnitSpec() {
             suspend { 1 },
             suspend { 2 },
             suspend { 3 }
-          ).traverse(::effectIdentity)
```

Comment:
`::identity`?

Reply:
`identity` doesn't work because it's not `suspend`

`effectIdentity` was giving me <img width="600" alt="Screen Shot 2019-06-12 at 8 43 02 AM" src="https://user-images.githubusercontent.com/772933/59352366-73896500-8cee-11e9-819a-3ff7a71d484b.png">